compiler/circle-opselector/src/OpSelector.cpp

```diff
@@ -156,6 +157,9 @@ std::unique_ptr<loco::Graph> make_graph(const std::vector<const luci::CircleNode
       }
     }
   }
+
+  const auto original_outputs = loco::output_nodes(g);
```

Comment:
The nodes already have its graph.
```suggestion
  const auto original_outputs = loco::output_nodes(const_cast<loco::Graph *>(nodes.at(0)->graph());
```